README.md

```diff
@@ -0,0 +1,8 @@
+# Alfresco Kubernetes Charts
+
+Source repository for *Alfresco Kubernetes Charts*
+
+## Disclaimer 
+
+As an end user of the charts, this is to inform you that for latest stable released version, use the charts in *stable* directory. 
+The charts in *incubator* are subject to change as we are overriding the version with every new release.
```

Comment:
I think I would change the order of the sentences to emphasize the warning about incubator, or some other way to get peoples attention on that.